box-content-sdk/src/main/java/com/box/androidsdk/content/BoxException.java

```diff
@@ -124,7 +125,7 @@ public BoxError getAsBoxError() {
      * @return a known error type that corresponds to a given response and code.
      */
     public ErrorType getErrorType() {
-        if (getCause() instanceof UnknownHostException) {
+        if (getCause() instanceof UnknownHostException || getCause() instanceof ConnectException) {
```

Comment:
What about in `BoxUtils.isConnectionIssueException`? Should we update the logic here and consolidate the code by using this Utils function?